.github/workflows/test-hawk.yaml

```diff
@@ -0,0 +1,51 @@
+name: e2e_test for hawk
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+env:
+  IP_NODE1: 172.17.0.2
+  IP_NODE2: 172.17.0.3
+
+jobs:
+  functional_test_crm_report_bugs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Start cluster nodes in containers
+        run: |
+          docker run -d --privileged -h node1 --name node1 --add-host node1:${IP_NODE1} --add-host node2:${IP_NODE2} ghcr.io/aleksei-burlakov/hawk-node
+          docker run -d --privileged -h node2 --name node2 --add-host node1:${IP_NODE1} --add-host node2:${IP_NODE2} ghcr.io/aleksei-burlakov/hawk-node
+      - name: Copy hawk to the nodes
+        run: |
+          cd ..
+          docker cp hawk node1:/
+          docker cp hawk node2:/
+      - name: Compile hawk in the nodes
+        run: |
+          docker exec -i node1 bash -c "cd hawk && make"
+          docker exec -i node2 bash -c "cd hawk && make"
+      - name: Install hawk in the nodes
+        run: |
+          docker exec -i node1 bash -c "cd hawk && make install && cp scripts/sysconfig.hawk /etc/sysconfig/hawk"
+          docker exec -i node2 bash -c "cd hawk && make install && cp scripts/sysconfig.hawk /etc/sysconfig/hawk"
+      - name: Start hawk
+        run: |
+          docker exec node1 systemctl start hawk hawk-backend
+          docker exec node2 systemctl start hawk hawk-backend
+      - name: Initialize container on the node1
+        run: docker exec node1 crm cluster init -u -n cluster1 -y
```

Comment:
@aleksei-burlakov crmsh init and join process should start hawk service if hawk.service is available
So, I think you could not use a specific step `Start hawk` to start hawk service